packages/manager/modules/cloud-universe-components/src/currency/service.js

```diff
@@ -28,4 +31,12 @@ export default class CucCurrencyService {
   getCurrentCurrency() {
     return this.currency;
   }
+
+  UcentsToCurrency(value, intervalParam) {
```

Comment:
```suggestion
  UcentsToCurrency(value, intervalParam = 1) {
```
instead ? 
Moreover we should probably don't have to do that on our side or it should be

Reply:
That's a good idea. Thanks. This is done